cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,56 @@
 /// <reference types='cypress' />
 
+import { user } from '../support/testData';
+
 describe('Book Store app', () => {
   before(() => {
-    
+    cy.visit('/login')
+  });
+
+  it('should provide an ability to login', () => {
+    cy.get('#userName').type(user.username);
+    cy.get('#password').type(user.password);
+    cy.get('#login').click();
+    cy.url().should('include', '/profile');
+    cy.contains('#submit', 'Log out').should('exist');
   });
 
-  it('', () => {
+  it('should privide an ability to add a book to collection', () =>{
+    cy.viewport(550, 750);
+    cy.login();
+    cy.visit('/profile');
     
+    cy.get('#gotoStore').click();
+    cy.get('.form-control').type('Speaking JavaScript');
+    cy.contains('Speaking JavaScript').click();
+    cy.get('#description-wrapper').should('exist');
+    cy.get('.text-right > #addNewRecordButton').click();
+    cy.on('window:alert', (str) => {
+        expect(str).to.equal(`Book added to your collection.`)
+    });
+    cy.visit('/profile');
+    cy.contains('Speaking JavaScript').should('exist');
+    cy.get('#delete-record-undefined').click();
+    cy.get('#closeSmallModal-ok').click();
+    cy.on('window:alert', (str) => {
+      expect(str).to.equal(`Book deleted.`)
+    });
+
+  });
+
+/*
```

Comment:
Please remove the commented-out code as it is not contributing to the test suite 🧹